src/components/FlatParameters/FlatParameters.js

```diff
@@ -326,13 +370,21 @@ export const FlatParameters = ({
                     return (
                         <>
                             <Select
+                                sx={{
+                                    ...(showSeparator && {
+                                        input: {
+                                            textAlign: 'right',
+                                            height: '10px',
```

Comment:
Why setting height this way, could we use `size` prop instead ?

Reply:
Done